Makefile

```
@@ -58,7 +58,7 @@ update-py:
 
 update-js:
 	# Install js packages
-	cd superset-frontend; npm install
+	cd superset-frontend; npm ci
```

A suggestion; instead of using `cd` do this:

```suggestion
	pushd superset-frontend; npm ci; popd
```

This will bring back the user to the directory they were when they ran the command.